backend/.env

```diff
@@ -0,0 +1,2 @@
+DB_USERNAME=""
+DB_PASSWORD=""
```

Comment:
Dalej nie mam przekonania czy to powinno być rozwiązane w taki sposób, że jest to w repo.